docs/portal/developer-portal/operations/UAN_Ansible_Roles.md

```diff
@@ -238,6 +276,27 @@ The default value of `uan_can_setup` is `no`.
 uan_can_setup: no
 ```
 
+`uan_can_bond_slaves`
+
+`uan_can_bond_slaves` is a list of the interfaces to use as the bond slave pair when `uan_can_setup` is true
+and the Customer Access Network (CAN) is configured on the system.
+
+The interface names must be in a format which doesn't change between reboots of the node, such as `ens10f1`
+which is the second port of the NIC in slot 10.  
+
+**NOTE:** `ens10f1` is typically the second port of the OCP 25Gb card and is used as one of the bond
+slaves in the CAN `bond0` interface.
+
+The second interface is typically the second port of a different 25Gb NIC for resiliency.

```

Comment:
So if we can bond the 1st 25Gb NIC of the first OCP card with the 1st 25Gb NIC of the second OCP card for NMN, and we can bond the 2nd 25Gb NIC of the first OCP card with the 2nd 25Gb NIC of the second OCP card for CAN, then theoretically, a customer could have both bonding configs at the same time?

Reply:
True.  This is what I did on surtur.  Bonded NMN on the first ports of two cards and bonded CAN on the second ports of those two cards.